packages/ckeditor5-image/src/imageinsert/imageinsertui.js

```diff
@@ -161,11 +161,7 @@ export default class ImageInsertUI extends Plugin {
 			const selectedElement = editor.model.document.selection.getSelectedElement();
 
 			if ( imageUtils.isImage( selectedElement ) ) {
-				editor.model.change( writer => {
-					writer.setAttribute( 'src', imageInsertView.imageURLInputValue, selectedElement );
-					writer.removeAttribute( 'srcset', selectedElement );
-					writer.removeAttribute( 'sizes', selectedElement );
-				} );
+				editor.execute( 'replaceImage', { image: selectedElement, source: imageInsertView.imageURLInputValue } );
```

Comment:
Do we actually need to pass the image here? Shouldn't the command take it from the selection?